[2021-04-09 14:56:35,462] tools.py->init_redis line:30 [INFO]rediscluster init success!
